aptos-move/framework/aptos-stdlib/sources/fixed_point64.move

```diff
@@ -112,7 +112,7 @@ module aptos_std::fixed_point64 {
     }
     spec create_from_rational {
         pragma opaque;
-        pragma timeout = 600; // Set larger timeout to avoid prover failure
+        pragma verify = false; // Proved, set to false because of timeout
```

Comment:
Please also add todo. as in `// TODO: set to false because of timeout (property proved).` This allow us to find this again easier.